README.md

```diff
@@ -14,9 +14,14 @@ pip install -r requirements.txt
 ```
 
 ## Usage
-To simply run the tool (notice the "x"):
+To simply run the tool (notice the "x" this is thenumber which we are unsure of):
 ```
-python wabf.py 6281288x272x7
+In Windows CMD
+    python3 wabf.py 6281288x272x7
+In Linux:
+    sudo chmod 777 wabf.py
+    python3 wabf.py
+     
```

Comment:
Good change! But I don't think there is a need to specify the command for each specific platform. I believe that it is kind of self-explanatory. Besides, there is no need for `sudo`-ing.